### PR TITLE
Add Suru background strip to desktop index page

### PR DESCRIPTION
**Pull request by MeltySnow**
## Done

Added suru strip to desktop index page.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/desktop
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Ensure that the design matches the [specification](https://github.com/canonical-web-and-design/web-squad/issues/1458)

## Issue / Card

Fixes canonical-web-and-design/web-squad#1458

### CODE REVIEW

File: static/sass/_pattern_strip.scss [hunk 1]
```diff
@@ -139,4 +140,31 @@
...
+  .p-strip--suru-background {
+    @extend %vf-strip;
+    background-blend-mode: multiply, normal, normal;
+    background-image:
+      linear-gradient(to bottom right, rgba(119, 41, 83, 0.16) 0%, rgba(119, 41, 83, 0.16) 49.9%, rgba(119, 41, 83, 0) 50%, rgba(119, 41, 83, 0) 100%),
+      linear-gradient(to top right, rgba(255, 255, 255, 1) 0%, rgba(255, 255, 255, 1) 49.6%, rgba(255, 255, 255, 0) 50%, rgba(255, 255, 255, 0) 100%),
+      linear-gradient(1.924rad, rgba(44, 0, 30, 1) 0%, rgba(119, 41, 83, 1) 28.6%, rgba(192, 82, 58, 1) 49.3%, transparent 49.4%);
+    background-position: left top, left 100.1%, left center;
+    background-repeat: no-repeat;
+    background-size: 100% 100%, 100% 17.4%, 100% 100%;
```
Comment (anthonydillon) [inline static/sass/_pattern_strip.scss]: 17.4% should be 4rem.

File: static/sass/_pattern_strip.scss [hunk 2]
```diff
@@ -139,4 +140,31 @@
...
   }
 }
+
+@mixin ubuntu-p-strip-suru-background {
+  .p-strip--suru-background {
+    @extend %vf-strip;
+    background-blend-mode: multiply, normal, normal;
+    background-image:
+      linear-gradient(to bottom right, rgba(119, 41, 83, 0.16) 0%, rgba(119, 41, 83, 0.16) 49.9%, rgba(119, 41, 83, 0) 50%, rgba(119, 41, 83, 0) 100%),
+      linear-gradient(to top right, rgba(255, 255, 255, 1) 0%, rgba(255, 255, 255, 1) 49.6%, rgba(255, 255, 255, 0) 50%, rgba(255, 255, 255, 0) 100%),
```
Comment (anthonydillon) [inline static/sass/_pattern_strip.scss]: I think `49.3%` is smoother then `49.6%`

File: static/sass/_pattern_strip.scss [hunk 3]
```diff
@@ -139,4 +140,31 @@
...
+      linear-gradient(1.924rad, rgba(44, 0, 30, 1) 0%, rgba(119, 41, 83, 1) 28.6%, rgba(192, 82, 58, 1) 49.3%, transparent 49.4%);
+    background-position: left top, left 100.1%, left center;
```
Comment (anthonydillon) [inline static/sass/_pattern_strip.scss]: Not sure we need the 100.1%. Cant see why its there?

File: static/sass/_pattern_strip.scss (hunk 1 above)
Comment (anthonydillon) [reply static/sass/_pattern_strip.scss]: Or is it `6rem` for a deep strip.

**Comment by anasereijo** [comment]
Looking awesome! Just some tiny things:

- [ ] could we move "millions of" to the second line of text?
I suggest using `<br class="u-hide--small u-hide--medium">`

- [ ] on 866px wide the text is overlaying the image
Screenshot 2019-07-31 at 16 15 40

- [ ] on 4K screens or very wide ones, the image gets smaller and should fill all the space:
Screenshot 2019-07-31 at 16 19 33

- [ ] on mobile the top and bottom paddings look very big. I know it's still 96px but could you check with @anthonydillon if this what it should do? I thought that the deep strips would change the paddings on smaller screens.

Screenshot 2019-07-31 at 16 21 30

**Comment by MeltySnow** [comment]
@anthonydillon I've changed the numbers to match your suggestions. The position was 100.1% because in Google Chrome there was a gap through which the solid gradient was  visible at the bottom, but changing the height to be in rem units seems to have fixed this.
image

**Comment by MeltySnow** [comment]
@anasereijo I've pushed an update that should hopefully address your issues.

**Comment by anasereijo** [comment]
Hi @MeltySnow,

Thanks for making those changes. It's looking great!

The only thing left to do is moving "millions of" to the 2nd line of text.

**Comment by MeltySnow** [comment]
@anasereijo I've pushed up the change we discussed earlier.

**Comment by anasereijo** [comment]
awesome!thanks

**Pull request merged**